### PR TITLE
Update ledger_entry method

### PR DESCRIPTION
**Pull request by mDuo13**
This PR updates the following things:

- Adds new fields from rippled 1.0 to the ledger_entry method docs
- Adds some potential error messages that weren't listed in the ledger_entry method's possible errors
- Fixes some bugs in the Checks example JavaScript that I discovered while testing ledger_entry.

I wanted to separate out the docs for all the different modes of ledger_entry, but after listing off how many modes there are (at least 10), I decided it was better to get this out first before embarking on a bigger restructure like that. I've created a task in the docs backlog for the bigger reworking of the ledger_entry docs ([DOC-1605 for those who have access to the RIpple JIRA](https://ripplelabs.atlassian.net/projects/DOC/issues/DOC-1605)).

### CODE REVIEW

File: content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md [hunk 1]
```diff
@@ -46,25 +46,33 @@ This method can retrieve several different types of data. You can select which t
...
 | `account_root`          | String - [Address][]       | _(Optional)_ Specify an [AccountRoot object](accountroot.html) to retrieve. |
-| `directory`             | Object or String           | _(Optional)_ Specify a [DirectoryNode](directorynode.html). (DirectoryNode objects each contain a list of IDs for things contained in them.) If a string, interpret as the [unique index](ledgers.html#tree-format) to the directory, in hex. If an object, requires either `dir_root` or `owner` as a sub-field, plus optionally a `sub_index` sub-field. |
+| `check`                 | String                     | _(Optional)_ Specify the [object ID](ledger-object-ids.html) of a [Check object](check.html) to retrieve. |
+| `directory`             | Object or String           | _(Optional)_ Specify a [DirectoryNode](directorynode.html). (DirectoryNode objects each contain a list of IDs for things contained in them.) If a string, must be the [object ID](ledger-object-ids.html) of the directory, as hexadecimal. If an object, requires either `dir_root` or `owner` as a sub-field, plus optionally a `sub_index` sub-field. |
```
Comment (jbheron) [inline content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md]: As discussed IRL, the use of 'things' could be changed.

**Review by jbheron — APPROVED** [review]
LGTM +1

(Made one comment)

File: content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md [hunk 2]
```diff
@@ -46,25 +46,33 @@ This method can retrieve several different types of data. You can select which t
...
-3. `directory` - Retrieve a [DirectoryNode](directorynode.html), which contains a list of other ledger objects
-4. `offer` - Retrieve an [Offer object](offer.html), which defines an offer to exchange currency
+3. `directory` - Retrieve a [DirectoryNode](directorynode.html), which contains a list of other ledger objects.
+4. `offer` - Retrieve an [Offer object](offer.html), which defines an offer to exchange currency.
 5. `ripple_state` - Retrieve a [RippleState object](ripplestate.html), which tracks a (non-XRP) currency balance between two accounts.
+6. `check` - Retrieve a [Check object](check.html), which is a potential payment that can be cashed by its recipient. [New in: rippled 1.0.0][]
+7. `escrow` - Retrieve an [Escrow object](escrow-object.html), which holds XRP until a specific time or condition is met. [New in: rippled 1.0.0][]
+8. `payment_channel` - Retireve a [PayChannel object](paychannel.html), which holds XRP for asynchronous payments. [New in: rippled 1.0.0][]
 
 If you specify more than one of the above items, the server retrieves only of them; it is undefined which it chooses.
```
Comment (scottschurr) [inline content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md]: Typo?  "only _one_ of them;"?

Comment (mDuo13) [reply content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md]: Nice! I think you just caught a typo that's been there at least 3 years. >_<

**Comment by scottschurr** [comment]
LGTM! :+1:

**Review by scottschurr — APPROVED** [review]
:+1:

**Pull request merged**